server/vcs/markdown/markdown_renderer.go

```diff
@@ -86,6 +86,20 @@ func (m *Renderer) Render(res command.Result, cmdName command.Name, baseRepo mod
 	return m.renderProjectResults(res.ProjectResults, common, baseRepo)
 }
 
+// RenderProject formats the data into a markdown string for a project
+// This method relies on the underlying methods used by Render() so skipping tests since it's indirectly tested using the existing tests for Render()
+func (m *Renderer) RenderProject(prjRes command.ProjectResult, cmdName command.Name, baseRepo models.Repo) string {
```

Comment:
How come we can't integrate this with Render? Like why is this method not called within Render?  We should try and dedup where we can.